ddtrace/contrib/django/middleware.py

```diff
@@ -18,6 +18,27 @@
 log = logging.getLogger(__name__)
 
 
+class TraceExceptionMiddleware(object):
```

Comment:
exactly, it's required to support the old Django middleware style. You can find more details here: https://docs.djangoproject.com/en/1.11/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware